lib/validator.js

```diff
@@ -47,8 +47,10 @@ class Validator {
     }
   }
 
-  validate (schemaRef, data) {
-    return this.ajv.validate(schemaRef, data)
+  validate (schemaRef, data, errors) {
+    const valid = this.ajv.validate(schemaRef, data)
+    if (this.ajv.errors && Array.isArray(errors)) errors.push(...this.ajv.errors)
```

Comment:
could this be faster?

```suggestion
    Array.isArray(errors) && Arrry.prototype.push.apply(errors, this.ajv.errors)
```